### PR TITLE
Migrate NATS module to java11

### PR DESCRIPTION
**Pull request by aashikam**
Fixes: https://github.com/ballerina-platform/module-ballerinax-nats/issues/33

## Purpose
$title

### CODE REVIEW

File: nats-ballerina/build.gradle [hunk 1]
```diff
@@ -1,295 +1,295 @@
...
+//    }
+//}
+//
+//unpackJballerinaTools.dependsOn copyToLib
+//unpackStdLibs.dependsOn unpackJballerinaTools
+//copyStdlibs.dependsOn unpackStdLibs
+//updateTomlVersions.dependsOn copyStdlibs
+//ballerinaBuild.dependsOn updateTomlVersions
+//ballerinaBuild.dependsOn ":nats-native:build"
+//build.dependsOn ballerinaBuild
```
Comment (daneshk) [inline nats-ballerina/build.gradle:295]: why are we commenting this `build.gradle` file.

Comment (aashikam) [reply nats-ballerina/build.gradle:295]: Fixed.

File: nats-native/src/main/java/module-info.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+module io.ballerina.stdlib.nats {
```
Comment (ThisaruGuruge) [inline nats-native/src/main/java/module-info.java:1]: Missing license headers

Comment (aashikam) [reply nats-native/src/main/java/module-info.java:19]: Fixed

**Review by daneshk — APPROVED** [review]

**Pull request merged**